mock/py/mockbuild/backend.py

```diff
@@ -179,17 +179,30 @@ def remove(self, *rpms):
         output = self.buildroot.pkg_manager.remove(*rpms, returnOutput=1)
         self.buildroot.root_log.info(output)
 
+    @traceLog()
+    def getPreconfiguredDeps(self, srpms):
```

Comment:
Can I ask here for docstring, please? I do not get from the name what preconfigured deps are.